src/components/ui/formik/validationSchemas.ts

```diff
@@ -105,7 +118,8 @@ export const vipValidationSchema = (
     is: (value) => !value,
     then: requiredOnceSet(initialValue)
       .concat(vipRangeValidationSchema(hostSubnets, values))
-      .concat(vipUniqueValidationSchema(hostSubnets, values)),
+      .concat(vipUniqueValidationSchema(hostSubnets, values))
+      .concat(vipRequired(values.hostSubnet)),
```

Comment:
`when` can be used instead of combining `concat`, `test` and using function to pass the hostSubnet value.
```suggestion
      .when('hostSubnet', {
        is: (value: string) => value !== NO_SUBNET_SET,
        then: Yup.string().required('Required. Please provide an IP address.'),
      }),
```

Reply:
done.